testdata/api_heartbeats_response_offline.json

```diff
@@ -0,0 +1,74 @@
+{
+    "responses": [
+        [
+            {
+                "data": {
+                    "branch": "heartbeat",
```

Comment:
The actual API only responds with the heartbeat's ID, no other attributes.

Reply:
All tests are expecting a heartbeat in the response. What's the payload returned?

Reply:
```json
{
 "responses": [
           [
               {
                   "data": {
                       "id": "845a922e-9e65-4775-bd68-bb3196d2e06a"
                   }
               },
               201
          ], ...
  ]
}
```

Reply:
@alanhamlett Im afk but I guess we don't parse the response correctly so we need to consider fixing it later in another PR.